data/transaction/status.go

```diff
@@ -9,18 +9,21 @@ import (
 type TxStatus string
 
 const (
-	// TxStatusReceived = received but not yet executed
-	TxStatusReceived TxStatus = "received"
-	// TxStatusPartiallyExecuted = received and executed on source shard
-	TxStatusPartiallyExecuted TxStatus = "partially-executed"
-	// TxStatusExecuted = received and executed
-	TxStatusExecuted TxStatus = "executed"
-	// TxStatusNotExecuted = received and executed with error
-	TxStatusNotExecuted TxStatus = "not-executed"
+	// TxStatusPending = received and maybe executed on source shard, but not on destination shard
+	TxStatusPending TxStatus = "pending"
+	// TxStatusSuccessful = received and executed
+	TxStatusSuccessful TxStatus = "successful"
```

Comment:
Breaking change for the exchanges consuming the API. But we will deal with in on the Proxy side somwhow :partying_face: 

Perhaps we can introduce a `v2` prefix in the URL on the Proxy (and re-map to old names in v1 URLs).

Reply:
added a jira ticket with ID `EN-7718`